contracts/ReversibleICO.sol

```diff
@@ -782,23 +863,26 @@ contract ReversibleICO is IERC777Recipient {
                     }
                 }
 
+                // return overflow tokens received
                 if(returnTokenAmount > 0) {
-                    // return overflow tokens received
-
-                    // allocate tokens to participant
+                    // send tokens back participant
```

Comment:
// send tokens back participant
>>
// send tokens back to participant

Reply:
fixed, thanks